Detectors/TRD/simulation/include/TRDSimulation/TrapSimulator.h

```diff
@@ -73,7 +73,7 @@ class TrapSimulator
   int getZeroSupressionMap(int iadc) const { return (mZSMap[iadc]); }
   bool isDataSet() { return mDataIsSet; };
   // set ADC data with array
-  void setData(int iadc, const ArrayADC& adc, std::vector<o2::MCCompLabel>& labels);
+  void setData(int iadc, const ArrayADC& adc, unsigned int digitIdx);
```

Comment:
From the source code, I find it difficult to understand what how the digit indices work. Can you document this, e.g. in the TDP before we forget how this was done?